### DISCUSSION TITLE
Senec + Evcc+Tibber

### DISCUSSION THREAD

**Discussion by KneterPeter** [marq24/ha-senec-v3#88] [Ideas]
Hi, ich suche nach einer Möglichkeit den Speicher nur die den Hausgebrauch zu nutzen und das Auto zu billigen Tibberpreisen komplett aus dem Netz zu laden. 

In der original Senecapp ist das unter schnelladen -> Experteneinstellungen möglich.

![Screenshot_20240415-201303.png](https://github.com/marq24/ha-senec-v3/assets/108238036/bb377164-203d-4101-892b-9088c0f6ae69)

Leider kann senec aber nicht mit tibber zusammen arbeiten. 

Über evcc kann ich mein Auto zu günstigen Zeiten über Ladeplanung laden aber da die wallbox dann über einen ETH/rs485 Adapter läuft denkt senec, dass ist ein großer Verbraucher und gibt die Energie aus dem Speicher frei. 

Über Https://ipdesSenecs//vars.html

Kann man auch Sachen einstellen. Nur weiß ich nicht wie senec das so regelt. 

Idee: den Wechselrichter drosseln auf den Bedarf des Hauses? Den kann man errechnen. Gesamtbetrag - Wallboxverbrauch.

Hat sich da schon jemand rangewagt? 

Wenn man den Leuten von evcc das auf dem Silbertablett präsentiert, können die das implementieren.

**Comment by marq24** [comment]
Entschuldige bitte die VIEL ZU SPÄTE Antwort - aber ich hatte bislang keine Benachrichtigungen bei neuen Diskussionen 

Also bei evcc habe ich ja inzwischen die ein oder andere Aktie drin... m.E. sollte/muss evcc die volle Kontrolle über die Wallbox-Steuerung haben - und das muss evcc auch intern/selber steuern können... [ob das evcc in der Zwischenzeit kann, habe ich natürlich keine Ahnung].

Ich kann mir aber nicht vorstellen, dass evcc die Option hätte, bei deinem SENEC the Speicher zu sperren... Und das eine Senec Wallbox die option hätte ausschließlich aus dem Netz zu laden ist auch eher unwahrscheinlich... 

Ich gebe aber auch gerne zu, dass ich Deine "Idee" _den Wechselrichter drosseln auf den Bedarf des Hauses? Den kann man errechnen. Gesamtbetrag - Wallboxverbrauch._ verstehe ich leider auch gar nicht... Also was Du meinst, wie man den Wechselrichter 'drosseln' können sollte... Also wenn überhaupt, dann müsste man den kompletten Speicherbezug abschalten... aber auch hier kenne ich keine option, wie das gehen sollte.

**Comment by KneterPeter** [comment]
Hi. Danke für die Antwort. Mittlerweile bin ich wieder weg von tibber und
bei Octopus Go. Damit lade ich im Winter spottbillig. Im Sommer hab ich
meine PV.
Evcc hat auch noch weitere Features bekommen und man kann tatsächlich den
Speicher über evcc aus dem Netz laden. Ich hab dir mal 2 Screenshots
angehangen.

Schönen Abend.

Matthias Marquardt ***@***.***> schrieb am Mo., 16. Juni
2025, 20:29:

**Reply by marq24** [reply]
leider kommen Attachment's beim eMailen nicht in github durch...
